### ISSUE TITLE
WebUI JSON parsing error with quotation mark in password

### ISSUE THREAD

**Issue by Tibladar** [Motion-Project/motionplus#105]
### Did you read the guide?

Yes

### What is the base version number of MotionPlus being used?

0.1.x

### What was the install method?

Built from source code

### What is base architecture?

x86/x64

### What is the distro being used?

Debian

### Disto version number

12

### Camera/Sound type(s) being used?

Network camera with RTSP

### Describe the issue/problem and steps to reproduce

Config strings aren't encoded for the WebUI JSON resulting in js errors:

![image](https://github.com/Motion-Project/motionplus/assets/52620063/70c2a061-d741-40f3-8281-fa4e338d1b1d)

Steps to reproduce:
- Add camera with a quotation mark in password 
 `,"netcam_userpass":{ "value":"username:herePass-->"<--word","enabled":true,"`
- Set `webcontrol_parms 2`
- Open WebUI
- See error in console

Either encode the string or throw an error to prevent using such a password.

Viewing and recording are working fine.

### Relevant MotionPlus log output (at log_level 8)

```shell
None

Browser:
 Uncaught SyntaxError: JSON.parse: bad escaped character at line 1 column 14612 of the JSON data
    onreadystatechange http://127.0.0.1:8080/:704
    initform http://127.0.0.1:8080/:702
    onload http://127.0.0.1:8080/:1
```

**Comment by Mr-Dave** [comment]
I thought this one was going to be straight forward but it turns out that I'm having difficulty replicating it with the current source.  

Can anyone validate it is still an issue with the code currently in the master branch?

**Comment by Mr-Dave** [comment]
closed with 17da8237ebec19ead503115516eeae6c5b0b3ab2
